src/styles/blog.css

```
@@ -0,0 +1,459 @@
+
+.blog-post__hero img {
+  display: block;
+  width: 100%;
+  height: auto;
+}
+
+.blog-prose {
+  width: min(100%, var(--blog-prose-width));
+  margin: var(--blog-space-6) auto 0;
+  @apply text-ctp-text;
+  font-size: 1.125rem;
```

Stylelint (17.5.0)

[error] Expected empty line before declaration (declaration-empty-line-before)

(declaration-empty-line-before)